sky/server/constants.py

```diff
@@ -3,7 +3,7 @@
 # API server version, whenever there is a change in API server that requires a
 # restart of the local API server or error out when the client does not match
 # the server version.
-API_VERSION = '1'
+API_VERSION = '2'
```

Comment:
minor: Is it possible that we only raise an update error when the user actually use `-u` with `sky jobs queue` or `sky jobs cancel`?

Reply:
I have no idea, it may require some pydantic investigation etc. Do we think this is quite valuable?

Reply:
It's fine for now, but once we have more user on client-server, it would be better to avoid frequent API version upgrade.